lib/src/header_parser/sub_parsers/objcinterfacedecl_parser.dart

```diff
@@ -106,35 +113,46 @@ void _parseProperty(clang_types.CXCursor cursor) {
   _logger.fine('       > Property: '
       '$fieldType $fieldName ${cursor.completeStringRepr()}');
 
-  final getter = ObjCMethod(
-    originalName: clang
-        .clang_Cursor_getObjCPropertyGetterName(cursor)
-        .toStringAndDispose(),
-    property: property,
-    dartDoc: dartDoc,
-    kind: ObjCMethodKind.propertyGetter,
-  );
-  getter.returnType = fieldType;
-  itf.addMethod(getter);
-
-  final setter = ObjCMethod(
-    originalName: clang
-        .clang_Cursor_getObjCPropertySetterName(cursor)
-        .toStringAndDispose(),
-    property: property,
-    dartDoc: dartDoc,
-    kind: ObjCMethodKind.propertySetter,
-  );
-  setter.returnType = NativeType(SupportedNativeType.Void);
-  setter.params.add(ObjCMethodParam(fieldType, 'value'));
-  itf.addMethod(setter);
+  final getterName =
+      clang.clang_Cursor_getObjCPropertyGetterName(cursor).toStringAndDispose();
+  if (config.objcMethods.shouldInclude(getterName)) {
+    final getter = ObjCMethod(
+      originalName: getterName,
+      name: config.objcMethods.renameUsingConfig(getterName),
```

Comment:
The declaration config also provides `renameMemberUsingConfig`. Can that be of any use?

In C it is used to rename the members of struct/unions

Reply:
Not sure. I separated objcInterfaces and objcMethods because I needed to be able to filter the methods, and there's no way of filtering members. Maybe it would be cleaner to add that though. Then there could just be one objcInterfaces config that we use to filter/rename both the interfaces and their methods

Reply:
Yeah, and it would probably be more consistent with structs/unions/enums, wdyt @dcharkes ?